embedded/drivers/include/analog.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __ANALOG_H__
+#define __ANALOG_H__
+#include <stdint.h>
+
+class Analog {
```

Comment:
I would change the class name to AnalogPin or similar

Reply:
got it